HHS/Head-Start-TTADP
# [TTAHUB-1583] Update "Child Incidents"

## Description of change

Change the reason "Child Incidents" to "Child Incident".

## How to test

- Run the migration make sure all reasons are updated.
- Create a report using the new Child Incident
- Search the filter using Child Incident.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1583


## Checklists

### Every PR



### Production Deploy


### After merge/deploy
